tasks/system_probe.py

```diff
@@ -851,6 +832,15 @@ def build_object_files(
     ninja_generate(ctx, nf_path, windows, major_version, arch, debug, strip_object_files, kernel_release)
     ctx.run(f"ninja -d explain -f {nf_path}")
 
+    if not is_root():
```

Comment:
I know this is how this block was written before, but I think it could be shorter with:
```
runner = ctx.run if is_root() else ctx.sudo
runner(f"...
```

Reply:
yes I had the same thought, will push this